### PR TITLE
Fix random body

### LINKED ISSUE #13901: Always Random Body isn't saving
(WEB REPORT BY: n3x15 REMOTE: 172.93.109.202:7777)
# Revision 
1852b5feb94cc2611f614b9950be30731466120c 
# Reproduction steps
1. Join server
2. Open character settings
3. Enable "Always Random Body"
4. Save Slot
5. Wait 30 seconds
6. Save Setup (Probably not necessary, but eliminating any weirdness)
7. Wait until server restart
8. Open character settings again

# Expected Behaviour
Always Random Body: Yes

# Observed Behaviour
Always Random Body: No

### PR DESCRIPTION
**Pull request by N3X15**
# Executive Summary
Implements the Always Random Save button's functionality.  While the button was there, it never actually worked and its state never saved.

# Failure Analysis
While the button was present and did set be_random_body, the code that actually triggered the appearance randomization was ordered incorrectly, taking place before stored appearance was set.  In addition, the method it used for randomization was just wrong.  Left as-is, it selects a random character slot from the old savefile system, which is deprecated.  Not only that, but the SQLite schema did not have a column to save the state of this button, nor the bindings to support it.  

In short, this thing was never implemented in the first place.

# Issues
* Fixes #13901

# Testing
Tested locally on BYOND 511.1385 on 6/24/2017

# Changelog
:cl:
- bugfix: Always Random Body button has now been completed and should work as expected.
/:cl:

### CODE REVIEW

**Comment by N3X15** [comment]
Secret repo MR made, see !7.
